README.md

```diff
@@ -2,23 +2,33 @@
 
 Here you can find configuration files and best practice instructions for deploying the 1Password SCIM Bridge on various public and private clouds.
 
-To streamline your setup, it is suggested to have Docker installed on your local machine.
 
-
-### Specific Deployment Guides
+## Automatic Deployment
 
 The easiest way to deploy the SCIM bridge is with our one-click installation currently available for Google Cloud Platform.  If you'd like to deploy to GCP [use this guide instead](https://support.1password.com/cs/scim-deploy-gcp/).
 
 We also have specific, detailed instructions for deployments to Azure using the [Azure Kubernetes Service](https://support.1password.com/cs/scim-deploy-azure/).
 
-### General Deployment Guides
 
-To deploy on Kubernetes, read the [Kubernetes example](https://github.com/1Password/scim-examples/tree/master/kubernetes)
+## Advanced Deployment
+
+The following guides are for anyone who doesn't want to use the one-click solution available on Google Cloud Platform.
```

Comment:
> doesn't want to use

Perhaps a softer wording of `is not using` would be better. Some infra and security requirements mean that some customers are unable to use a 1-click, and so it is not really a choice.

Reply:
Ditto re DO 1-click also existing.